cleanup for howtos3 (#638)
openedx/docs.openedx.org: source/educators/how-tos/update_rerun_course.rst

>   .. only:: Partners

Reviewer: Actually, delete everything under `.. only:: Partners`, please.

Can you please check the other files in this pull request to see if they have anything specified as `.. only:: Partners`?

Author: done! I've reviewed the 27 files and no further reference for that.

Reviewer 2: I think the ".. only::" comment still needs to be deleted, yes?

Author: Done

  .. only:: Partners

>     Another example of a user interface change is for courses on the `Open edX Website <https://openedx.org/>`_

Reviewer: ```suggestion
```
Just remove this line

    A new **Help** option was added to the LMS in September 2016 to

>     provide access to the *EdX Learner's Guide*. When this option was added,

Reviewer: ```suggestion
    provide access to the *Open edX Learner's Guide*. 
```

>     the previous **Help** option was renamed **Support**.

Reviewer: ```suggestion

```
